molgenis-data-validation/src/main/java/org/molgenis/data/validation/EntityAttributesValidator.java

```diff
@@ -39,7 +40,8 @@ public EntityAttributesValidator(ExpressionValidator expressionValidator)
 
 	public Set<ConstraintViolation> validate(Entity entity, EntityType meta)
 	{
-		Set<ConstraintViolation> violations = checkValidationExpressions(entity, meta);
+		Set<ConstraintViolation> violations = checkNullableExpressions(entity, meta);
```

Comment:
Filling in a nullable expression will currently break the usability of the forms (required indication+validation / required-only button). I assume we consider the EMX format change a toggle? "As long as you don't use this, the client won't break"?

Reply:
Without the front-end changes the form will only be validated server-side, not client-side. The front-end work is scheduled as part of the next release.